### DISCUSSION TITLE
Raspberry Pi 4 – docker: no matching manifest for linux/arm/v8 in the manifest list entries.

### DISCUSSION THREAD

**Discussion by terraboss** [nextcloud/all-in-one#3408] [Questions] [answered]
Again and again. 

I don't understand why I can't just install Nextcloud AIO! Such errors should be caught. Or at least for popular platforms, like the Raspberry Pi, there should be at least installation instructions that address these kind of problems and offer solution steps.

**docker: no matching manifest for linux/arm/v8 in the manifest list entries.**

Client: Docker Engine - Community
 Version:    24.0.6
 Context:    default

Kernel Version: 6.1.21-v8+
 Operating System: Raspbian GNU/Linux 11 (bullseye)
 OSType: linux
 Architecture: aarch64

**Comment by szaimen** [comment, marked as answer]
Hi, AIO needs a 64-bit OS in order to work.

**Reply by terraboss** [reply]
Raspberry Pi OS is 64 Bit. 

But it seems, my fault is, I've picked the wrong one in the list. (Full, 32-bit)

**Reply by MichiStig** [reply]
i have the same problem, how did you solve it? did you pick the 32-bit version?

**Reply by terraboss** [reply]
Indeed, I accidentally selected the 32-bit version. The 64-bit version works fine.

**Reply by superduper11** [reply]
What list are you guys talking about?

I entered the command on Step 3 of "https://github.com/nextcloud/all-in-one" and it is returning the exact same error message.
The CPU architecture of my machine is "aarch64" so I don't see the issue here.

**Reply by dani4ko03** [reply]
I am also wondering what list is being referred to? Is it the selection of the OS when using Raspberry Pi Flasher? because I am having the same issue and I am pretty sure I picked the 64bit image.

**Reply by naivewen666** [reply]
hi there, what list is being referred to? in your original post, your OS is already aacrch64, right?

**Comment by liukuo362573** [comment]
I encounter the same problem,"no matching manifest for linux/arm/v8 in the manifest list entries
",the issue is that my linux is aarch64,the image is arm64,it is ok,but i install docker is arm.

**Reply by naivewen666** [reply]
I have the same issue.
